src/highrl/callbacks/robot_callback.py

```diff
@@ -301,12 +302,17 @@ def run_n_episodes(
         DataFrame: Logs for evaluation episodes.
     """
     env.opt.episode_statistics["scenario"] = "robot_env_test"
+    i = 0
     for _ in range(num_eposides):
         obs = env.reset()
         done = False
+
         while not done:
             action, _ = model.predict(obs, deterministic=True)
             obs, _, done, _ = env.step(action)
+            _LOG.debug("test episode %s", i)
```

Comment:
Since `i` is an integer, it should be output as an integer `%i` not a string `%s`.